examples/cli/cli_demo.go

```diff
@@ -55,9 +55,9 @@ func connect() {
 	//config.Log = log.New(os.Stdout, "", log.Ldate|log.Ltime|log.Lshortfile)
 	config.Log = infoLogger
 	config.Log.SetPrefix("PubNub :->  ")
-	config.PublishKey = "demo"
-	config.SubscribeKey = "demo"
-	config.SecretKey = "demo"
+	config.PublishKey = "pub-c-c1648ded-d156-4a2d-9dbb-a23262945fe2"
```

Comment:
I assume we shouldn't be including these keysets in here?

Reply:
removed